src/app/wizard/step/cluster/component.ts

```diff
@@ -193,6 +198,10 @@ export class ClusterStepComponent extends StepBase implements OnInit, ControlVal
       .valueChanges.pipe(takeUntil(this._unsubscribe))
       .subscribe(() => (this._clusterSpecService.admissionPlugins = this.form.get(Controls.AdmissionPlugins).value));
 
+    this.control(Controls.CNIPlugin).valueChanges.subscribe(() => {
+      this.updateCNIPluginOptions();
+    });
```

Comment:
Please change to one-liner:

```ts
this.control(Controls.CNIPlugin).valueChanges.subscribe(_ => this.updateCNIPluginOptions());
```

Also, add unsubscribe (see other subscriptions for examples).